spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/http/GsonHttpMessageConvertersConfiguration.java

```diff
@@ -41,7 +43,7 @@
 
 	@Configuration
 	@ConditionalOnBean(Gson.class)
-	@Conditional(PreferGsonOrMissingJacksonCondition.class)
+	@Conditional(PreferGsonOrMissingJacksonAndJsonbCondition.class)
```

Comment:
regarding to the conditional thing. Should I rewrite conditions with an `SpringBootCondition` implementation? Just to make sure, I would like to know that the current conditions are ok.